test/test_pf_to_vf.py

```diff
@@ -26,9 +26,35 @@ def test_pf_to_vf_lb_tcp(prepare_ifaces, grpc_client):
 		f"Wrong packet received (ip: {dst_ip}, dport: {dport})"
 
 	grpc_client.dellbtarget(lb_name, lbpfx_ul_ipv6)
-	grpc_client.dellbprefix(VM1.name, lb_ip)
+	grpc_client.dellbprefix(VM1.name, f"{lb_ip}/32")
```

Comment:
Two forgotten places that should now use `lb_pfx`

Reply:
Done. There was also a similar usage with "/128" lb_ip6 which I corrected as well.